src/single-spa-vue.js

```diff
@@ -75,6 +79,12 @@ function bootstrap(opts, mountedInstances, props) {
 function mount(opts, mountedInstances, props) {
   const instance = {};
   return Promise.resolve().then(() => {
+    // It ensures "clean start" for every Vue app mount
+    // Otherwise it would first attempt to render route that was rendered before unmount
+    if (opts._startRoute) {
+      opts.appOptions.router.history.updateRoute(opts._startRoute);
```

Comment:
Does it a documented method? If not - there should be a check that it really exists.

Reply:
Looks like it's not documented